worlds/messenger/options.py

```diff
@@ -139,7 +139,9 @@ class MusicBox(DefaultOnToggle):
 
 
 class NotesNeeded(Range):
-    """How many notes are needed to access the Music Box."""
+    """
+    How many notes need to be found in order to access the Music Box. 6 are always needed to enter, so this places the others in your start inventory.
```

Comment:
You used this kind of wording earlier, unsure if you'd want to do that here as well (totally optional)
```suggestion
    How many notes need to be found/received in order to access the Music Box. 6 are always needed to enter, so this places the others in your start inventory.
```

Reply:
Could we argue that "receiving" a music note in this case could be construed as having occurred when the notes are placed in start inventory?